src/DistributedMetadata/DDLService.h

```diff
@@ -31,7 +31,7 @@ class DDLService final : public MetadataService
     Int32 sendRequest(const String & payload, const Poco::URI & uri, const String & method, const String & query_id) const;
     Int32 doDDL(const String & payload, const Poco::URI & uri, const String & method, const String & query_id) const;
     void createTable(IDistributedWriteAheadLog::RecordPtr record);
-    void mutateTable(IDistributedWriteAheadLog::RecordPtr record, const String & method) const;
+    void mutateTable(IDistributedWriteAheadLog::RecordPtr record, String method) const;
```

Comment:
we don't need to change the signature. I was thinking if we can introduce more `OpCodes` like `ADD_COLUMN`, `DELETE_COLUMN` etc? In this way, column operation is clearly separated from table mutation which includes data mutation.

Reply:
fixed